vcpkg.json

```diff
@@ -26,7 +26,10 @@
     "clib-util",
     "unordered-dense",
     "efsw",
-    "tracy"
+    {
+      "name": "tracy",
+      "version>=": "0.11.0"
```

Comment:
Also needs to update baseline, I did this by doing the following:
- run `git clone` in local vcpkg repo
- run `vcpkg x-update-baseline` in local CS repo. This will update `builtin-baseline` in vcpkg.json
- manually update `VCPKG_COMMIT_ID` to be the same in `.github/workflows/build.yaml`
- when doing this I got a build error for imgui, so we might need to pin it (or fix that compile error)